gemini/use-cases/retrieval-augmented-generation/utils/intro_multimodal_rag_utils.py

```diff
@@ -702,22 +738,25 @@ def print_text_to_image_citation(
         )
 
         # Print the cosine similarity score
-        print(color.BLUE + f"score: " + color.END, image_dict["cosine_score"])
+        print(color.BLUE + "score: " + color.END, image_dict["cosine_score"])
```

Comment:
I think the `Color` class could be removed and replaced with the `colorama` library.